lib/topological_inventory/schema/base.rb

```diff
@@ -21,6 +26,20 @@ def self.klass_for(name)
         # checks whether it loads what we want
         klass if klass.to_s == klass_name
       end
+
+      private
+
+      def link_data_to_refresh_state_part
```

Comment:
@slemrmartin I wonder if this makes more sense in the topological_inventory-persister?  Thinking maybe in this [method](https://github.com/RedHatInsights/topological_inventory-persister/blob/master/lib/topological_inventory/persister/workflow.rb#L87-L98)